src/wp-admin/widgets-form.php

```diff
@@ -187,9 +187,9 @@
 
 	// Remove old position.
 	if ( ! isset( $_POST['delete_widget'] ) ) {
-		foreach ( $sidebars_widgets as $sidebar_id => $sidebar ) {
-			if ( is_array( $sidebar ) ) {
-				$sidebars_widgets[ $sidebar_id ] = array_diff( $sidebar, array( $widget_id ) );
+		foreach ( $sidebars_widgets as $loop_sidebar_id => $loop_sidebar ) {
+			if ( is_array( $loop_sidebar ) ) {
+				$sidebars_widgets[ $loop_sidebar_id ] = array_diff( $loop_sidebar, array( $widget_id ) );
```

Comment:
The array variable is `$sidebars_widgets`, so maybe the other variables could use the singular form of that.

```suggestion
		foreach ( $sidebars_widgets as $sidebar_widget_id => $sidebar_widget ) {
			if ( is_array( $sidebar_widget ) ) {
				$sidebars_widgets[ $sidebar_widget_id ] = array_diff( $sidebar_widget, array( $widget_id ) );
```

I ran a search for `$sidebar_widget` and only found `$sidebar_widget_ids` (plural form, in `WP_Customize_Manager` and `WP_Customize_Widgets`).

I also thought to use `$sidebar_index`, but that variable name is already defined in another part of `widgets-form.php`.